dashboard/app/controllers/application_controller.rb

```diff
@@ -231,6 +231,9 @@ def prevent_caching
       response[:puzzle_ratings_enabled] = script_level && PuzzleRating.can_rate?(script_level.script, level, current_user)
     end
 
+    # We no longer set options[:activity] (see https://github.com/code-dot-org/code-dot-org/pull/63713),
+    #   so response[:activity_id] will always be nil.
+    # TODO: figure out whether anything depends on activity_id being set, and delete this line if not.
```

Comment:
Open to feedback on whether you think it's worth doing this as part of this PR or not.

Reply:
I lean yes. It should just be in the frontend code that calls milestone_response, correct? Even if it is trying to access `activity_id`, then it'll get undefined, which it's currently already getting, right?

Reply:
Yeah, good call. It ended up being easier than I expected it to be! Note that this does still represent a *slight* change of behavior in that we were formerly getting `null` but will now get `undefined`, but fortunately we were only attempting to reference this value from one place in frontend code anyway, and that difference does not matter there.